kilkari/src/main/java/org/motechproject/nms/kilkari/domain/SubscriptionPack.java

```diff
@@ -14,13 +13,19 @@
     @Field
     private String name;
 
-    public SubscriptionPack(String name) {
+    @Field
+    private SubscriptionPackType type;
 
-        this.name = name;
+    public SubscriptionPack() {
     }
 
-    public SubscriptionPack() {
+    public SubscriptionPack(String name) {
```

Comment:
Should we allow this constructor?  It seems like a type should always be provided.